### PR TITLE
Release

### PR DESCRIPTION
**Pull request by b-gehrke**

### CODE REVIEW

File: Upper Level BCIO/bcio_upper_level.owl [hunk 1]
```diff
@@ -746,7 +747,7 @@
     <!-- http://humanbehaviourchange.org/ontology/BCIO_045000 -->
 
     <owl:Class rdf:about="http://humanbehaviourchange.org/ontology/BCIO_045000">
-        <rdfs:subClassOf rdf:resource="http://purl.obolibrary.org/obo/OBI_0000011"/>
```
Comment (jannahastings) [inline Upper Level BCIO/bcio_upper_level.owl:749]: A few lines up, the class 'style of delivery', which actually hasn't changed in this release in this file, now needs to change status. It is still listed as 'proposed' (along with one or two other related classes). However, the style of delivery spreadsheet has recently been updated and all classes in it have now got curation status 'published'. That means that these classes, also represented here, are getting double curation statuses in the merged file.  This is an other example of something that in the long term we would solve pre-release. However, for now could you update the status in this one to 'published' ?

File: BehaviourChangeTechniques/bcto.owl [hunk 2]
```diff
@@ -1,42 +1,26 @@
...
+     xmlns:foaf="http://xmlns.com/foaf/0.1/"
      xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#"
      xmlns:terms="http://purl.org/dc/terms/"
      xmlns:schema="https://schema.org/"
      xmlns:Property="https://www.wikidata.org/wiki/Property:"
-     xmlns:oboInOwl="http://www.geneontology.org/formats/oboInOwl#">
-    <owl:Ontology rdf:about="http://humanbehaviourchange.org/ontology/bcto.owl">
+     xmlns:oboInOwl="http://www.geneontology.org/formats/oboInOwl#"
+     xmlns:owlready_ontology="http://www.lesfleursdunormal.fr/static/_downloads/owlready_ontology.owl#">
+    <owl:Ontology rdf:about="http://humanbehaviourchange.org/ontology/../BehaviourChangeTechniques/bcto.owl">
```
[the remark this answers is not in the document]
Comment (b-gehrke) [reply BehaviourChangeTechniques/bcto.owl:17]: I don't know either. We did not import it explicitly but it occurs already in `bcio_external.owl`. Same for `Property`. But the BFO class `entity` has a  `Property:P5191` annotation which itself has a `owlready_ontology:python_name` annotation. Maybe that's why robot/the OWLAPI added it?

File: Upper Level BCIO/bcio_upper_level.owl (hunk 1 above)
Comment (b-gehrke) [reply Upper Level BCIO/bcio_upper_level.owl:749]: I don't think I understand. The "BCI style of delivery" is marked as "Published" and in the final owl file I also see only one curation status annotation.

Comment (jannahastings) [reply Upper Level BCIO/bcio_upper_level.owl:749]: It was 'communication style' not 'BCI style of delivery'. I think it is the parent? 

Screenshot 2023-11-21 at 11 31 44

These  three classes were potentially affected by this.  
Screenshot 2023-11-21 at 11 32 36

Comment (b-gehrke) [reply Upper Level BCIO/bcio_upper_level.owl:749]: Should be fixed now.

FYI: Just changing it to proposed and rerun the release script still produced both "Proposed" and "Published". The issue was that the subontologies import `http://humanbehaviourchange.org/ontology/bcio_upper_level.owl` and in the final merge command robot resolves imports relative to each input file. But as the new `bcio_upper_level.owl` is not in the same directory as the subontologies but rather in the `scripts` directory, robot downloads the upper level ontology from the web and hence uses an older version which still contained the `'has curation status' Proposed`.

File: Upper Level BCIO/bcio_upper_level.owl [hunk 3]
```diff
@@ -34,7 +35,7 @@
     <!-- http://humanbehaviourchange.org/ontology/BCIO_001000 -->
 
     <owl:Class rdf:about="http://humanbehaviourchange.org/ontology/BCIO_001000">
-        <rdfs:subClassOf rdf:resource="http://purl.obolibrary.org/obo/OBI_0000011"/>
+        <rdfs:subClassOf rdf:resource="http://purl.obolibrary.org/obo/COB_0000082"/>
```
[the remark this answers is not in the document]
Comment (b-gehrke) [reply Upper Level BCIO/bcio_upper_level.owl:38]: I don't know what we can do about that. SEPIO, which we import, defines both `OBI_0000011` and `COB_0000082`with the same label `planned process`.

File: BehaviourChangeTechniques/bcto.owl [hunk 4]
```diff
@@ -1,42 +1,26 @@
 <?xml version="1.0"?>
-<rdf:RDF xmlns="http://humanbehaviourchange.org/ontology/bcto.owl#"
-     xml:base="http://humanbehaviourchange.org/ontology/bcto.owl"
+<rdf:RDF xmlns="http://humanbehaviourchange.org/ontology/../BehaviourChangeTechniques/bcto.owl#"
```
[the remark this answers is not in the document]
Comment (b-gehrke) [reply BehaviourChangeTechniques/bcto.owl]: Done

**Pull request merged**